sdk/mixedreality/mixedreality-authentication/package.json

```diff
@@ -112,6 +112,18 @@
     "typedoc": "0.15.2"
   },
   "//sampleConfiguration": {
-    "skipFolder": true
+    "skipFolder": true,
+    "productName": "Azure Mixed Reality Authentication",
+    "productSlugs": [
+      "azure",
+      "object-anchors",
+      "remote-rendering",
+      "spatial-anchors"
```

Comment:
I'm not 100% sure on what to do here. This particular package is shared among the 3 services i've listed here, but I don't know if the right thing is to include these product slugs or if I should just exclude them entirely.

@witemple-msft Thoughts?

Reply:
We should use all of them, that way if someone is just browsing the sample programs for Spatial Anchors, these samples will also show up. Someone who's using that product for any reason may also be interested to know that there are samples for mixed reality authentication.

That said, these should all be prefixed with `azure` (e.g. `azure-spatial-anchors`), and I don't believe `azure-object-anchors` (or any Object Anchors slug) exists yet. We'll need to get in touch with the docs platform to get that one added to the taxonomy.

Reply:
@witemple-msft How do we go about getting the `azure-object-anchors` slug created? That looks to be the last thing blocking my PR. I suppose I could remove it and add it back later.

Reply:
@craigktreasure We have a handful of missing product slugs, and I'll write this one down and request for it to be added along with the rest of the ones we need.. Let's just remove it for now and I'll let you know or just add it myself when it's available.